packages/iris-grid/src/sidebar/visibility-ordering-builder/sortable-tree/utilities.ts

```diff
@@ -0,0 +1,345 @@
+import type { UniqueIdentifier } from '@dnd-kit/core';
+import { arrayMove } from '@dnd-kit/sortable';
+import {
+  GridUtils,
+  ModelSizeMap,
+  MoveOperation,
+  ModelIndex,
+} from '@deephaven/grid';
+import type IrisGridModel from '../../../IrisGridModel';
+import type ColumnHeaderGroup from '../../../ColumnHeaderGroup';
+import type { FlattenedItem, TreeItem, TreeItems } from './types';
+
+export const iOS = /iPad|iPhone|iPod/.test(navigator.platform);
```

Comment:
Should be named `isIOS`

Reply:
Not needed since we don't do any mobile stuff